crm/modules/member/install.inc.php

```diff
@@ -160,4 +160,13 @@ function member_install($old_revision = 0) {
             }
         }
     }
+    if ($old_revision < 7) {
+        // Alter plan table to rename columns
+        $sql = '
+            ALTER TABLE plan CHANGE active enabled tinyint(1);
+            ALTER TABLE plan CHANGE voting active tinyint(1);
+        ';
+        $res = mysqli_query($db_connect, $sql);
+        if (!$res) crm_error(mysqli_error($db_connect));
```

Comment:
If install is not called upon initial run this will not get hit.
There should be a way force a change like this, given that we are modifying the structure of the database. We should hook this into starting up so those that don't know to run install get where they need to be.
I tried running install on my initial testing and it did not go well.

Reply:
Additionally,  i was told explicitly to leave docker out of production commits.
I have it all set up and ready to be dropped in when we want to test it.

Reply:
The only way is to manual use the "upgrade" feature of the crm. Depending on the change you may have to manually browse to the /upgrades page but there's no automatic upgrade currently. We can discuss if we want this feature for the future but that's out of scope for this fix

Reply:
What docker? there aren't any dockerfiles here. The only thing I did was add .gitattributes so that docker files aren't added during a merge.